scripts/ci/ensure-staged-sass.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash
 
+###
+# Copyright 2017 resin.io
```

Comment:
Haha, I had to double-check and indeed this file was only created in 2017! Feels like much longer ago ;-)